versionist.conf.js

```diff
@@ -0,0 +1,59 @@
+var execSync = require('child_process').execSync;
+
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
```

Comment:
Any reason for using this here, when the 'upstream' [versionist.conf.js](https://github.com/resin-io/versionist/blob/master/versionist.conf.js) does:
```
const execSync = require('child_process').execSync;

const getAuthor = (commitHash) => {
  return execSync(`git show --quiet --format="%an" ${commitHash}`, {
    encoding: 'utf8'
  }).replace('\n', '');
};
```

IMHO if any differences can be minimised, that'd be ideal.
At the moment, I'm aware of `resin-base`, `edge-node-manager` and `leech` all using their own `versionist.conf.js` files. See also https://github.com/resin-io/versionist/issues/84